libraries/SITL/SIM_AirSim.cpp

```diff
@@ -0,0 +1,165 @@
+/* 
+	Simulator Connector for AirSim
+*/
+
+#include "SIM_AirSim.h"
+
+#include <stdio.h>
+#include <arpa/inet.h>
+#include <errno.h>
+
+#include <AP_HAL/AP_HAL.h>
+
+extern const AP_HAL::HAL& hal;
+
+using namespace SITL;
+
+AirSim::AirSim(const char *home_str, const char *frame_str) :
+	Aircraft(home_str, frame_str),
+	sock(true),
+	opened_socket(false)
+{
+	printf("Starting SITL Airsim\n");
+}
+
+/*
+	Bind in/out socket
+*/
+bool AirSim::bind_socket(void)
+{
+	if (opened_socket) {
+		return true;
+	}
+
+	if (!sock.bind("0.0.0.0", airsim_sensor_port)) {
```

Comment:
why not use set_interface_ports() here ? it will allow you to select the port you want

Reply:
That wasn't used in the other backends, so didn't use it in this one also.
Now checked the Gazebo files, will use it
Thanks for the review!!

Reply:
I can make this change but then this will set the ports to 9002 & 9003 which are different from the default ports used by Airsim. The ports can be set through `settings.json`

Just confirming once before making the changes

Reply:
Done